### ISSUE TITLE
How to search hyperparameters for a new model?

### ISSUE THREAD

**Issue by ashok-arjun** [ShiyuLiang/odin-pytorch#7]
Hi - thanks for your work.

I would like to use your method on another architecture.

I see that there are three hyperparameters:

1. Perturbation magnitude (epsilon)
2. Temperature (T)
3. Threshold (delta)

How should I choose these hyperparams using the OOD validation set? 

I understand that the OOD validation set should be different from the ones used for testing, but I'm unsure which of these hyperparams I should prioritise. 

Should I choose do a coarse grid search for all of these?

Thank you!

**Comment by YixuanLi** [comment]
Hi Arjun,

I recommend checking out our latest work on energy score (https://arxiv.org/abs/2010.03759) - which is hyperparameter-free and gives comparable results to ODIN. 

-Sharon

**Comment by ashok-arjun** [comment]
Thank you @YixuanLi !
